website/docs/installation/ferretdb/kubernetes.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 4
+---
+
+# Kubernetes
+
+To deploy FerretDB on Kubernetes, you need to have a running Kubernetes cluster and a PostgreSQL instance with the DocumentDB extension.
+Please see the [DocumentDB installation docs](../documentdb/kubernetes.md) for more information on how to deploy it on Kubernetes.
+
+We provide different FerretDB images for various deployments.
+Please see the [Docker installation docs](docker.md) to learn more on the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `2.1.0`)
+to ensure consistency across deployments.
+Ensure to [enable telemetry](../../telemetry.md) to receive notifications on the latest versions.
+
+For more information on the best DocumentDB version to use, see the [corresponding release notes for the FerretDB version](https://github.com/FerretDB/FerretDB/releases/).
+:::
+
+With a running Kubernetes cluster and a PostgreSQL instance with the DocumentDB extension, create a `ferretdb.yaml` file with the following content:
```

Comment:
Funny how there we create a cluster, but file creates an instance on the other page